documentation/pages/move-registry/tooling/typescript-sdk.mdx

```diff
@@ -0,0 +1,86 @@
+import { Tabs } from 'nextra/components'
+import { Callout } from 'nextra/components'
+
+# Typescript SDK plugin
+
+The purpose of the MVR TypeScript SDK plugin is to offer developers a seamless experience when building PTBs with MVR names. 
```

Comment:
Sorry about my noob question but what is an SDK plugin?

Reply:
Check more info here: 
https://sdk.mystenlabs.com/typescript/plugins